core-metastore/src/main/java/org/zalando/nakadi/repository/db/EventTypeRepository.java

```diff
@@ -109,7 +109,8 @@ public EventType mapRow(final ResultSet rs, final int rowNum) throws SQLExceptio
                         "FROM zn_data.event_type," +
                         "jsonb_to_recordset(et_event_type_object->'authorization'->'writers') " +
                         "AS writers(data_type text, value text) " +
-                        "WHERE writers.data_type = ? AND writers.value = ?",
+                        "WHERE writers.data_type = ? AND writers.value = ? " +
+                        "ORDER BY et_name",
```

Comment:
can we have it as an option in the api? `sort_by=X`, X is event type name, owning_application etc

Reply:
Not sure that brings a lot of value.  Filter by `owning_application` would be more appreciated, I guess.